SECURITY.md

```diff
@@ -0,0 +1,7 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report a Vulnerability with the built-in Github functionality in the *Security* tab : https://github.com/geniusyield/atlas-docs/security/advisories
```

Comment:
Need to correct the url from `atlas-docs`.

Reply:
Done